src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -109,5 +109,21 @@ inline arm_compute::DataLayout getAclDataLayoutByMemoryDesc(MemoryDescCPtr desc)
     return arm_compute::DataLayout::UNKNOWN;
 }
 
+/**
+* @brief Custom move dimensions for specific NHWC layout
+* @param _listDims List of dimensions
+*/
+inline void dimensionsMover(const std::vector<arm_compute::TensorShape*> &_listDims) {
```

Comment:
Do I understand correctly that this function moves channel dimension to 0th index in case nspc layout provided?
If yes shouldn't we use it for all ACL executors, not only Eltwise and Deconv?